### ISSUE TITLE
API for Chrome

### ISSUE THREAD

**Issue by jan-runkel-zeitgleich** [mujx/hakatime#6]
It would be great if there was an Api endpoint for the current user. 
I have adapted the Chrome Extension, but unfortunately the endpoint is missing :-(
Is there a plan to provide this endpoint?

https://domain.com/api/v1/users/current

**Comment by mujx** [comment]
Yes, that's something that could be implemented, at least in a stripped down version. The [docs](https://wakatime.com/developers#users) of the original endpoint have some fields that don't make sense for hakatime (e.g `has_premium_features`, `is_hireable` etc)

**Comment by luckydonald** [comment]
`has_premium_features: true`  😈
